src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="card__star"></div>
```

Comment:
the name od the block is stars, not card

```suggestion
      <div class="stars__star"></div>
```